akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/Dialect.scala

```diff
@@ -109,23 +108,70 @@ private[projection] object DialectDefaults {
  * INTERNAL API
  */
 @InternalApi
-private[projection] case class DefaultDialect(schema: Option[String], tableName: String) extends Dialect {
+private[projection] case class H2Dialect(schema: Option[String], tableName: String, lowerCase: Boolean)
+    extends Dialect {
 
-  def this(tableName: String) = this(None, tableName)
+  def this(tableName: String, lowerCase: Boolean) = this(None, tableName, lowerCase)
 
-  private val table = schema.map(s => s""""$s"."$tableName"""").getOrElse(s""""$tableName"""")
+  def transform(stmt: String): String =
+    if (lowerCase) stmt.toLowerCase
```

Comment:
maybe use `akka.util.Helpers.toRootLowerCase` ?
it was introduced due to some problem with toLowerCase when using Turkish locale